float64 xCor
float64 yCor
float64 width
float64 height
float64 prob
string className
string actionName
